airflow/providers/google/cloud/hooks/datafusion.py

```diff
@@ -386,15 +480,29 @@ def start_pipeline(
             pipeline_name,
             "workflows",
             "DataPipelineWorkflow",
-            "start"
+            "start",
         )
+        runtime_args = runtime_args or {}
+        # Unfortunately making the start call to CDAP does not return a run_id to poll for state.
+        # So we are adding a faux job id.
+        job_id = str(uuid.uuid4())
```

Comment:
It would be easier for the user to understand what started a job (in the CDAP UI) if this contained the airflow task id or dag run id.

Reply:
Then the id will have to be generated in the operator. From hook level we are unable (in a simple way) access this information.

Reply:
Could the hook parameterize this faux id to let the operator do this?

Reply:
Done